images.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Image struct {
+	Name string
+	Tags []Tag
+}
+
+type Tag struct {
+	Sha string
+	Tag string
+}
+
+var Images = []Image{
+	Image{
+		Name: "quay.io/coreos/hyperkube",
+		Tags: []Tag{
+			Tag{
+				Sha: "5ff22b5c65d5b93aa948b79028dc136a22cda2f049283103f10bd45650b47312",
+				Tag: "v1.5.2_coreos.2",
+			},
+		},
+	},
+}
```

Comment:
We talked conceptually. Retagger handles third party images moving to quay. We may need to think about how to synchronise images to local registries in the future.